lib/glossarist/concept.rb

```diff
@@ -73,6 +75,14 @@ def default_designation
       localized&.terms&.first&.designation
     end
 
+    # All Related Concepts
+    # @return [Array<RelatedConcept>]
+    attr_reader :related
+
+    def related=(related)
+      @related = related.map { |r| RelatedConcept.new(r) }
```

Comment:
What happens if the object passed in here is already a RelatedConcept object? Preferably the new just returns the object itself.

Reply:
I've updated the `new` method to return the object if it is already initialized.